compiler/regex.go

```diff
@@ -125,7 +127,7 @@ func CompileRegex(tpl string, delimiterStart, delimiterEnd byte) (*regexp.Regexp
 		// Build the regexp pattern.
 		varIdx := i / 2
 		fmt.Fprintf(pattern, "%s(%s)", regexp.QuoteMeta(raw), patt)
-		varsR[varIdx], err = regexp.Compile(fmt.Sprintf("^%s$", patt))
+		varsR[varIdx], err = regexp2.Compile(fmt.Sprintf("^%s$", patt), regexp2.None)
```

Comment:
Why did you choose `None` here? Wouldn't that make it backwards incompatible?

Reply:
Yes, thank you. Switched to another option.